src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -200,6 +201,21 @@ export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true,
       onSelectedDatasetChange(approvedDatasets);
     }, [datasets]);
 
+    // Required to get Chairperson acknowledgments of closeouts
+    useEffect(() => {
+      async function fetchAcknowledgements() {
+        if (readOnlyMode) {
+          const acknowledgementMap: AcknowledgementMap = await User.getAcknowledgements();
```

Comment:
You can `get` a single acknowledgement:
/api/user/acknowledgements/{key}

You know the key already:
dar_closeout_chair_ref_'${dar.referenceId}'